tests/executeAutomagically.spec.ts

```diff
@@ -53,6 +53,33 @@ describe(executeAutomagically.name, () => {
     expect(core.getInput).toHaveBeenCalledWith('environmentName')
   })
 
+  it('includes variablesToOverwrite name if defined', async () => {
+    const variablesToOverwrite = ['key1:value1', 'key2:value:2']
```

Comment:
maybe an explicit test for the value containing ':'s with a title explaining it-> would help in understanding i think